azavea/idb-osm-extraction-tool
# Create map and sidebar components

## Overview

This PR begins the app by adding map and sidebar components, styled roughly to match the wireframes.

I also started some Sass components, including one with color variables which I extracted from the wireframes.


40429c6 drops redux-act and immutability-helper, which we won't be using in the project.

Connects #2 

### Demo

![screen shot 2018-07-16 at 3 20 30 pm](https://user-images.githubusercontent.com/4165523/42778652-d29a5d5c-890b-11e8-8776-d9a4f5622e35.png)

## Testing Instructions

- get, build, and serve this branch, then visit http://localhost:4567 and ensure the app loads as depicted above in both Chrome and IE11
